### DISCUSSION TITLE
Can't understand some tracy's results

### DISCUSSION THREAD

**Discussion by Tonaion02** [bevyengine/bevy#14445] [Q&A]
Hello guys, i am using tracy to profile my application.
I have obtained some strange results during profiling.

For example if we put attention on this spans:
![image](https://github.com/user-attachments/assets/ba8302b3-e4e3-4ec4-ae54-fb5aa2aa5193)
The system_task call system. But the mean elapsed time of system_task is less than mean elapsed time of system like we can see in this screenshots: 
![image](https://github.com/user-attachments/assets/4aff3bba-e5b4-4314-a76e-c54140071b6e)
![image](https://github.com/user-attachments/assets/6ef602f7-c5fd-4d2d-acc2-0b4959bd7aa2)
So the elapsed mean time of the caller is less than the elapsed time for the function called? Is it possible? I lost something on how bevy works?

**Comment by hymm** [comment]
System task is an async task and has instances where it doesn't run the system, but is doing some other work to coordinate with the system executor.

**Reply by Tonaion02** [reply]
So it's something normal that i can ignore?
You can indicate me the part of the doc where i can read something about that?

**Reply by hymm** [reply]
Yeah, you should just ignore the system task spans. It's more useful for internal engine devs to see how the executor is behaving than something the end user should care about. There's a good chance they get removed at some point, since you're not the only one who's been confused by them.

No real doc. I've been meaning to write a blog post about how to interpret tracy logs, but who knows when I'll get around to it.
